packages/e2e-tests/src/specs/wordpress/postLocking.js

```diff
@@ -33,9 +33,6 @@ const percyCSS = `.dashboard-grid-item-date { display: none; }`;
 const storyTitle = 'Test post lock';
 
 describe('Post Locking', () => {
-  withExperimentalFeatures(['enablePostLocking']);
-  withPlugin('e2e-tests-post-lock-mock');
```

Comment:
For some reason, running this before the `beforeAll` on line 38 made the story locked _before_ the story was even created.

Moving this to line 44 adds the `beforeAll` from `withPlugin` after the `beforeAll` on line 39. Now the test creates the story that will be tested on _before_ turning on the plugin.

This doesn't change the behavior of the test, but it does ensure that the story can be created prior to the tests.

Reply:
Changes are in https://github.com/GoogleForCreators/web-stories-wp/pull/10901

Reply:
Going to remove my changes here 😄